test/parallel/test-net-dns-lookup.js

```diff
@@ -29,12 +29,12 @@ const server = net.createServer(function(client) {
   server.close();
 });
 
-server.listen(0, '127.0.0.1', common.mustCall(function() {
+server.listen(0, common.localhostIP, common.mustCall(function() {
   net.connect(this.address().port, 'localhost')
```

Comment:
```suggestion
  net.connect(this.address().port, common.localhostIP)
```
Probably `localhost` also resolves to IPv4 first in the CI test. So I guess we need to change it.